app/config/config.neon

```diff
@@ -148,6 +148,13 @@ parameters:
       solutionPointsUpdatedPrefix: "Assignment Points Updated - "
       shadowPointsUpdatedPrefix: "Shadow Assignment Points Awarded - "
 
+  generalStatsNotifications:
+    period: "1 week"
+    emails:
+      from: %emails.from%
+      to: %failures.emails.to%
```

Comment:
This is rather unexpected dependency on other specific configuration parameter, I would suggest to establish new general parameter with name for example `emails.adminEmail` or similar and use this one in `failures` and in here.